{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/test.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from {{cookiecutter.repo_name}}.settings.base import *
+from {{cookiecutter.repo_name}}.settings.utils import get_logger_config
+
+
+# TEST SETTINGS
+INSTALLED_APPS += (
+    'django_nose',
+)
+
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+NOSE_ARGS = [
+    '--with-ignore-docstrings',
+    '--logging-level=DEBUG',
+]
+
+LOGGING = get_logger_config(debug=False, dev_env=True, local_loglevel='DEBUG')
+
+
+class DisableMigrations(object):
+    """Override method calls on the MIGRATION_MODULES dictionary.
+
+    If the `makemigrations` command has not been run for an app, the
+    `migrate` command treats that app as unmigrated, creating tables
+    directly from the models just like the now-defunct `syncdb` command
+    used to do. These overrides are used to force Django to treat apps
+    in this project as unmigrated.
+
+    Django 1.8 features the `--keepdb` flag for exactly this purpose,
+    but we don't have that luxury in 1.7.
```

Comment:
Since this repo's README advertises the use of Django 1.8, I think it'd be worth doing away with this 1.7 hack now and taking advantage of the `--keepdb` flag.